core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -241,7 +246,7 @@ angular.module('oppia').component('contributionsAndReview', {
           size: 'lg',
           resolve: {
             suggestionIdToSuggestion: function() {
-              return angular.copy(suggestionIdToSuggestion);
+              return cloneDeep(suggestionIdToSuggestion);
```

Comment:
Curious what the difference is?

Reply:
It is the same except angular.copy does not work with Angular 2+